crates/compiler/test_mono/src/tests.rs

```diff
@@ -3289,3 +3289,150 @@ fn non_nullable_unwrapped_instead_of_nullable_wrapped() {
         "#
     )
 }
+
+#[mono_test]
+#[ignore = "Hits an unimplemented for abilities, not sure why..."]
```

Comment:
Is this still the case?

Reply:
Yes, still fails, also fails in a similar test for encode. Haven't dug into it much:
```
thread 'inspect_custom_type' panicked at '[Abilities] not yet implemented. Tracking issue: https://github.com/roc-lang/roc/issues/2463.
Additional information: Not reachable yet', crates/compiler/mono/src/layout.rs:511:55
```